projects/1.18/assets/sebastrnlib/sebastrnlib/lang/zh_cn.json

```
@@ -0,0 +1,47 @@
+{
+  "item.sebastrnlib.book": "Sebastrn的模组!",
+  "sebastrnlib.landing": "你好$(playername)！$(br2)将你的$(6)懒人厨房(CFB)$()烹饪桌，连接到你最喜爱的网络存储模块。$(br2)$(4)精致厨房$()连接$(6)CFB$()到$(1)精致存储$().$(br2)$(2)应用厨房$()连接$(6)CFB$()到$(5)应用能源2$().",
+
+
+  "category.refinedcooking.name": "精致厨房",
+  "category.refinedcooking.description": "一个允许您使用$(1)精致存储$()直接在您的$(6)懒人厨房$()的厨房方块工作！$(br2)两个新方块和一个物品被添加到Minecraft。$(br2)如果您下载了mod，你会看到右边的章节/条目。如果没有，您也可以下载$(l:https://www.curseforge.com/minecraft/mc-mods/refined-cooking)here$()",
+
+  "category.appliedcooking.name": "应用厨房",
+  "category.appliedcooking.description": "一个允许您使用$(5)应用能源2$()直接在您的$(6)懒人厨房$()的厨房方块工作！$(br2)一个新方块被添加到Minecraft。$(br2)如果您下载了mod，你会看到右边的章节/条目。如果没有，您也可以下载$(l:https://www.curseforge.com/minecraft/mc-mods/applied-cooking)here$()",
+
+  "category.meloncube.name": "MelonCube Promo",
+  "category.meloncube.description": "单击右侧的条目以获取有关Minecraft托管服务惊人折扣的更多信息！",
+
+
+  "entry.refinedcooking.kitchenstation.name": "厨房终端",
+  "entry.refinedcooking.kitchenstation.page0": "厨房终端是您需要连接$(6)CFB$()到$(1)精致存储$()的方块之一。将其视为$(1)RS$()网络接收器。$(br2)一旦你拥有了它，你就需要把它作为你的$(6)CFB$()厨房方块，用厨房网卡右键单击它，然后将该卡放入连接到您的$(1)RS$()网络。",

```

这一类条目的现状译法都有些奇怪。下面的类似条目也需要改进一下翻译。
```suggestion
  "entry.refinedcooking.kitchenstation.page0": "厨房终端是您连接$(6)CFB$()到$(1)精致存储$()所需的方块之一。其功能相当于$(1)RS$()网络接收器。$(br2)一旦你拥有了它，你就需要把它作为你的$(6)CFB$()厨房方块，用厨房网卡右键单击它，然后将该卡放入连接到您的$(1)RS$()网络的厨房网络接入点。",
```